p4-fusion/git_api.cc

```diff
@@ -318,7 +336,7 @@ std::string GitAPI::WriteChangelistBranch(
 		GIT2(git_signature_new(&author, authorName.c_str(), authorEmail.c_str(), cl.timestamp, timezoneMinutes));
 
 		// -3 to remove the trailing "..."
-		std::string commitMsg = cl.number + " - " + cl.description + "\n[p4-fusion: depot-paths = \"" + depotPath.substr(0, depotPath.size() - 3) + "\": change = " + cl.number + "]";
+		std::string commitMsg = std::to_string(cl.number) + " - " + cl.description + "\n[p4-fusion: depot-paths = \"" + depotPath.substr(0, depotPath.size() - 3) + "\": change = " + std::to_string(cl.number) + "]";
```

Comment:
Generally one should avoid string concatentation with `+`. Instead use `std::stringstream` or `std::format` (since C++20) or add a dependency on `fmt` (https://fmt.dev/latest/usage.html#usage-with-cmake).